docs/walkthroughs/local-setup.adoc

```diff
@@ -141,6 +141,22 @@ used to point to a local running MCP server rather than the default of
 looking up a Route names `mcp-standalone` and using that as the MCP
 server host.
 
+=== UI Style guide
```

Comment:
@sedroche I think this would live better in the top level README rather than down in the local setup guide.

Reply:
Ah ok. Misunderstanding. Will move it.